client/jetpack-onboarding/steps/woocommerce.jsx

```diff
@@ -22,6 +25,12 @@ class JetpackOnboardingWoocommerceStep extends React.PureComponent {
 
 		return (
 			<Fragment>
+				<DocumentHead title={ translate( 'WooCommerce ‹ Jetpack Onboarding' ) } />
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Homepage ‹ Jetpack Onboarding' )` *ES Score: 9*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/jetpack-onboarding-pageview-tracker)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).